scripts/globals/spells/trust/ajido-marujido.lua

```diff
@@ -27,10 +27,20 @@ function onMobSpawn(mob)
 
     mob:addSimpleGambit(ai.t.TARGET, ai.c.NOT_SC_AVAILABLE, 0,
                         ai.r.MA, ai.s.HIGHEST, tpz.magic.spellFamily.NONE, 30)
+
+    tpz.trust.synergyMessage(mob, {
```

Comment:
I expect using the term "synergy" will be confusing once we have synergy craft skill work started.